### PR TITLE
Small fixes in Ubuntu docs

### PR DESCRIPTION
**Pull request by sevein**
Small changes after feedback received from one of our users - thank you!

It needs to be ported to `2.4`.

### CODE REVIEW

File: admin-manual/installation/linux/ubuntu-trusty.rst [hunk 1]
```diff
@@ -45,6 +48,13 @@ During the installation, you will be prompted to set a password for the default
 administrator user (root). We strongly recommend that you use a strong password
 and please write it down as you are going to need it later.
 
+.. warning::
+
+   If you are planning to use MySQL 5.7 please be aware that it has not been
+   fully tested yet and we know at least of `one bug <https://projects.artefactual.com/issues/9051>`__
+   that it has not been solved yet! MySQL 5.6 will work fine but it is not
```
Comment (fiver-watson) [inline admin-manual/installation/linux/ubuntu-trusty.rst]: don't need the "it" here - should be: 

> we know at least of `one bug <https://projects.artefactual.com/issues/9051>`__
> that has not yet been solved! MySQL 5.6 will work fine but it is not

File: admin-manual/installation/linux/ubuntu-xenial.rst [hunk 2]
```diff
@@ -30,20 +33,28 @@ Install the dependencies
...
+better than its previous major release in terms of speed, scalability and
 user-friendliness. Also, we've experienced very good results using forks like
 Percona Server or MariaDB, so don't be afraid and use them if you want!
 
-Let's install MySQL using :command:`apt install`:
+.. warning::
+
+   If you are planning to use MySQL 5.7 please be aware that it has not been
+   fully tested yet and we know at least of `one bug <https://projects.artefactual.com/issues/9051>`__
+   that it has not been solved yet!
```
Comment (fiver-watson) [inline admin-manual/installation/linux/ubuntu-xenial.rst:45]: same as above comment

File: admin-manual/installation/linux/ubuntu-xenial.rst [hunk 3]
```diff
@@ -30,20 +33,28 @@ Install the dependencies
 MySQL
 -----
 
-We strongly recommend using `MySQL <https://www.mysql.com/>`__  5.6 as it's
-much better than its previous major release in terms of speed, scalability and
+We strongly recommend using `MySQL <https://www.mysql.com/>`__  5.6 as it's much
+better than its previous major release in terms of speed, scalability and
```
Comment (fiver-watson) [inline admin-manual/installation/linux/ubuntu-xenial.rst]: "better than the previous major release" works better than "its previous major release

File: admin-manual/installation/linux/ubuntu-trusty.rst (hunk 1 above)
Comment (fiver-watson) [reply admin-manual/installation/linux/ubuntu-trusty.rst]: re: line 36 - in the Xenial instructions this is updated to recommending MySQL 5.6 - do we need to make changes for the Trusty docs too? What comes bundled, and is this why the difference?

**Comment by sevein** [comment]
Thank you! :***

**Comment by sevein** [comment]
Also in `2.4`!

**Pull request merged**